dev/src/codewind/project/Project.ts

```diff
@@ -129,6 +130,8 @@ export default class Project implements vscode.QuickPickItem {
         // appImageLastBuild is a string
         this._lastImgBuild = new Date(Number(projectInfo.appImgLastBuild));
 
+        this._autoInjectMetricsEnabled = projectInfo.injectMetrics;
```

Comment:
Can you initialize this to `false`, just in case it's missing form the `projectInfo` we don't end up with it `undefined`?

`this._autoInjectMetricsEnabled = projectInfo.injectMetrics || false`

Reply:
I've made this change.